backend/FwLite/MiniLcm/SyncHelpers/WritingSystemSync.cs

```diff
@@ -5,15 +5,15 @@ namespace MiniLcm.SyncHelpers;
 
 public static class WritingSystemSync
 {
-    public static async Task<int> Sync(WritingSystems currentWritingSystems,
-        WritingSystems previousWritingSystems,
+    public static async Task<int> Sync(WritingSystems previousWritingSystems,
```

Comment:
while we're at it lets change these from `previous` to before, and `current` to after for all the WS sync code

Reply:
Done in commit af5dfe7c.